akka-server-app/src/main/scala/de/mukis/AkkaServerApp.scala

```diff
@@ -0,0 +1,11 @@
+package de.mukis
+
+import akka.actor.{ ActorSystem, Props }
+import scala.concurrent.duration._
+import com.typesafe.config.ConfigFactory
+
+object AkkaServerApp extends App{
+  val config = ConfigFactory.load
```

Comment:
Do you mean the application.conf with the Akka and custom settings? This would be definitifly an improvement. This costs me an hour last week :)

The application.conf file is specified with the -Dconfig.file=path-to-file option and this option is specified with JAVA_OPT... in the etc-default file. But I will search for a solution. I would be nice if the server program for example understand the option -c path-to-config file.